tcf_website/static/landing/landing-page.css

```diff
@@ -0,0 +1,192 @@
+/*!
+ * Start Bootstrap - Landing Page v5.0.8 (https://startbootstrap.com/themes/landing-page)
+ * Copyright 2013-2020 Start Bootstrap
+ * Licensed under MIT (https://github.com/StartBootstrap/startbootstrap-landing-page/blob/master/LICENSE)
+ */
+
+ /* tCF Style Classes */
+:root {
+ --main-color: #254aa1;
+ --secondary-color: #db6b1a;
+}
+.text-indigo {
+ color: #254aa1;
+}
+.bg-indigo {
+ background-color: #254aa1;
+}
+
+/* Utility classes */
+.link-unstyled, .link-unstyled:link, .link-unstyled:hover {
+ color: inherit;
+ text-decoration: inherit;
+}
+
+/* Html Classes */
+body {
+  font-family: 'Lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-weight: 700;
+}
+
+/* Navbar */
+nav.navbar {
+  background-color: white;
+}
+
+/* Masthead */
+header.masthead {
+  position: relative;
+  /* background-color: #343a40; */
+  /* background: url("../about/team-pfps/bg-tCF-2017-team.png") no-repeat center center; */
+  /* background-size: cover; */
+  padding-top: 8rem;
+  padding-bottom: 0rem;
+}
+
+/* Black translucent overlay */
+/* header.masthead .overlay {
+  position: absolute;
+  background-color: #212529;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  left: 0;
+  opacity: 0.3;
+} */
```

Comment:
Needs to be removed eventually if you don't plan on keeping it

Reply:
done